ddl/db_test.go

```diff
@@ -2004,6 +2007,229 @@ func (s *testDBSuite1) TestCreateTable(c *C) {
 	c.Assert(err.Error(), Equals, "[types:1291]Column 'a' has duplicated value 'B' in ENUM")
 }
 
+func (s *testDBSuite6) TestRepairTable(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test")
+	s.tk.MustExec("drop table if exists t")
+	s.tk.MustExec("drop table if exists other_table")
+	s.tk.MustExec("drop table if exists origin")
+
+	// Test repair table when TiDB is not in repair mode.
+	s.tk.MustExec("CREATE TABLE t (a int primary key, b varchar(10));")
+	err := domain.GetDomain(s.s).Reload()
+	c.Assert(err, IsNil)
+	_, err = s.tk.Exec("admin repair table t CREATE TABLE t (a float primary key, b varchar(5));")
+	c.Assert(err, NotNil)
+	c.Assert(err.Error(), Equals, "[ddl:8215]Failed to repair table: TiDB is not in REPAIR MODE")
```

Comment:
Can we use `MustGetErrCode`?

Reply:
Now `admin repair table` only use one ErrCode `ErrRepairTable` with different args string.